spring-cloud-kubernetes-examples/kubernetes-hello-world-example/pom.xml

```diff
@@ -9,7 +9,7 @@
 	</parent>
 	<modelVersion>4.0.0</modelVersion>
 
-	<artifactId>kubernetes-hello-world</artifactId>
+	<artifactId>kubernetes-hello-world-example</artifactId>
```

Comment:
rename the module, we use one artifactId here, but in modules declaration of the parent pom we use a different name, as such when I try to do `-pl '-:kubernetes-hello-world'` or `-pl -:kubernetes-hello-world-example` to exclude this one from the build, maven is really confused and says that neither is present. A simple rename fixes this.